pwsh-extension-dev.code-workspace

```diff
@@ -0,0 +1,493 @@
+          "TestE2EPowerShell",
+        ],
+        "default": "Clean"
+      }
+    ],
+
+  },
+  "launch": {
+    "version": "0.2.0",
+    "compounds": [
+      {
+        "name": "Launch Extension",
+        "configurations": [
+          "Launch",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 1
+        }
+      },
+      {
+        "name": "Launch Extension - Temp Profile",
+        "configurations": [
+          "Launch-Temp",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "PreLaunch",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 2
+        }
+      },
+      {
+        "name": "Launch Extension - Isolated Profile",
+        "configurations": [
+          "Launch-Isolated",
+          "PowerShell Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 3
+        }
+      },
+      {
+        "name": "Interactively Test Rename",
+        "configurations": [
+          "Launch Extension - Rename Test Cases",
+          "Attach to Editor Services"
+        ],
+        "preLaunchTask": "Build",
+        "stopAll": true,
+        "presentation": {
+          "hidden": false,
+          "group": "Test",
+          "order": 4
+        }
+      }
+    ],
+    "configurations": [
+      {
+        "name": "Launch",
+        "type": "extensionHost",
+        "request": "launch",
+        "env": {
+          "VSCODE_PARENT_SESSION_ID": "${command:GetVsCodeSessionId}",
+        },
+        "runtimeExecutable": "${execPath}",
+        "args": [
+          "--extensionDevelopmentPath=${workspaceFolder:Client}",
+          "${workspaceFolder:Client}/examples"
+        ],
+        "sourceMaps": true,
+        // This speeds up source map detection and makes smartStep work correctly
+        "outFiles": [
+          "${workspaceFolder:Client}/**/*.js",
+          "!**/node_modules/**",
+          "!**/.vscode-test/**"
+        ],
+        "skipFiles": [
+          "<node_internals>/**",
+          "**/node_modules/**",
+          "**/.vscode-test/**",
+          "**/app/out/vs/**" //Skips Extension Host internals
+        ],
+        "presentation": {
+          "hidden": true
+        }
+      },
+      {
+        "name": "Launch-Temp",
+        "type": "extensionHost",
+        "request": "launch",
+        "env": {
+          "VSCODE_PARENT_SESSION_ID": "${command:GetVsCodeSessionId}",
+        },
+        "runtimeExecutable": "${execPath}",
+        "args": [
+          // Runs the extension in an empty temp profile that is automatically cleaned up after use
+          // Undocumented: https://github.com/microsoft/vscode-docs/issues/6220
```

Comment:
Is it _still_ undocumented?

Reply:
Pretty much, I should just make a PR at this point.

Temporary profiles are sort of covered here: https://code.visualstudio.com/docs/editor/profiles#_what-is-a-temporary-profile but it's not mentioned anywhere you can use --profile-temp in the docs, only in github issues and comment replies that I've seen.